<launch>

	<node pkg="visual_behavior_los_ultramarinos" type="nodo_camera_vb" name="nodo_camera_vb" output="screen">
	</node>

	<node pkg="visual_behavior_los_ultramarinos" type="ObservadorPersonas" name="ObservadorPersonas" output="screen">
	</node>

	<node pkg="visual_behavior_los_ultramarinos" type="ObservadorPelotasHSV" name="ObservadorPelotasHSV" output="screen">
	</node>

	<node pkg="visual_behavior_los_ultramarinos" type="Coordinador" name="Coordinador" output="screen">
	</node>

	<node pkg="visual_behavior_los_ultramarinos" type="Control" name="Control" output="screen">
	</node>

</launch>
